nptdms/tdms.py

```diff
@@ -145,6 +145,17 @@ def groups(self):
 
         return list(self._groups.values())
 
+    def has_group(self, group_name):
```

Comment:
This seems like a good use case for implementing the `__contains__` method, so rather than using `tdms_file.has_group('group_name')` you could use `'group_name' in tdms_file`, which I think would be more idiomatic Python.

In fact it looks like that second syntax already works due to `TdmsFile` and `TdmsGroup` implementing the `__iter__` method, but it probably makes sense to provide a more efficient implementation.

Reply:
Thanks for the comment. 
I agree I will see if I can get it working either today or Saturday :)